tbx/images/migrations/0002_initial.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.2.8 on 2023-12-22 16:53
```

Comment:
Query: why do some apps have more than one initial migration?

Reply:
Yes, this feels a little strange. We should be able to have just a single one for most apps. Maybe deleting and regenerating again will resolve this?

Reply:
Thanks, will try this. these were generated automatically via `makemigrations`, I don't know why Django created two migrations. It may be related to the new `WagtailImageField` which was introduced in [Wagtail 4.2](https://docs.wagtail.org/en/stable/releases/4.2.html#wagtail-specific-image-field-wagtailimagefield). According to the release notes, this will generate a new migration. Looking into it ...

Reply:
@ababic wen I remove all the migrations, and attempt to recreate them, I am still getting multiple migrations for some apps:

```console
tbx@52a0da8acbd3:/app$ ./manage.py makemigrations --dry-run

Migrations for 'blog':
  tbx/blog/migrations/0001_initial.py
    - Create model BlogIndexPage
    - Create model BlogIndexPageRelatedLink
    - Create model BlogPage
    - Create model BlogPageAuthor
    - Create model BlogPageRelatedLink
    - Create model BlogPageTagSelect
  tbx/blog/migrations/0002_initial.py
    - Add field tag to blogpagetagselect
    - Add field link_document to blogpagerelatedlink
    - Add field link_page to blogpagerelatedlink
    - Add field page to blogpagerelatedlink
    - Add field author to blogpageauthor
    - Add field page to blogpageauthor
    - Add field feed_image to blogpage
    - Add field related_services to blogpage
    - Add field social_image to blogpage
    - Add field link_document to blogindexpagerelatedlink
    - Add field link_page to blogindexpagerelatedlink
    - Add field page to blogindexpagerelatedlink
    - Add field social_image to blogindexpage
Migrations for 'events':
  tbx/events/migrations/0001_initial.py
    - Create model Event
    - Create model EventIndexPage
  tbx/events/migrations/0002_initial.py
    - Add field social_image to eventindexpage
  tbx/events/migrations/0003_initial.py
    - Add field author to event
    - Add field page to event
    - Add field related_services to event
Migrations for 'images':
  tbx/images/migrations/0001_initial.py
    - Create model CustomImage
    - Create model Rendition
Migrations for 'impact_reports':
  tbx/impact_reports/migrations/0001_initial.py
    - Create model ImpactReportAuthor
    - Create model ImpactReportPage
  tbx/impact_reports/migrations/0002_initial.py
    - Add field author to impactreportauthor
    - Add field page to impactreportauthor
Migrations for 'navigation':
  tbx/navigation/migrations/0001_initial.py
    - Create model NavigationSettings
Migrations for 'people':
  tbx/people/migrations/0001_initial.py
    - Create model ContactReasonsList
    - Create model CulturePage
    - Create model ValuesPage
    - Create model ValuesPageValue
    - Create model PersonPage
    - Create model PersonIndexPage
    - Create model CulturePageLink
    - Create model CulturePageKeyPoint
    - Create model ContactReason
    - Create model Contact
    - Create model Author
Migrations for 'taxonomy':
  tbx/taxonomy/migrations/0001_initial.py
    - Create model Service
Migrations for 'torchbox':
  tbx/core/migrations/0001_initial.py
    - Create model Advert
    - Create model HomePage
    - Create model JobIndexPage
    - Create model ParticleSnippet
    - Create model Tag
    - Create model StandardPage
    - Create model SocialMediaSettings
    - Create model MainMenu
    - Create model JobIndexPageJob
    - Create model HomePageHeroImage
    - Create model HomePageFeaturedPost
    - Create model GlobalSettings
    - Create model AdvertPlacement
Migrations for 'work':
  tbx/work/migrations/0001_initial.py
    - Create model WorkPage
    - Create model WorkPageTagSelect
    - Create model WorkPageScreenshot
    - Create model WorkPageAuthor
    - Create model WorkIndexPage
```

Reply:
Maybe I should do it separately for each app ... let me see what happens if I do that

Reply:
Yep! That seems to work! :tada:

Reply:
Thanks! testing again...